doc/plugin-guide.md

```diff
@@ -46,7 +46,7 @@ Please refer to the [HTTP instrumentation][http-plugin] or [gRPC instrumentation
 
 [shimmer]: https://github.com/othiym23/shimmer
 [builtin-plugins]: https://github.com/open-telemetry/opentelemetry-js#plugins
-[base-plugin]: https://github.com/open-telemetry/opentelemetry-js/blob/master/packages/opentelemetry-core/src/trace/instrumentation/BasePlugin.ts#L29
+[base-plugin]: https://github.com/open-telemetry/opentelemetry-js/blob/e9b2cf9aeb1daf5ffbab800681bfe1cafc636576/packages/opentelemetry-core/src/platform/node/BasePlugin.ts#L30
```

Comment:
this is wrong you should not point to specific version but to a master branch, just find the correct location - it was moved

Reply:
Thanks for catching this